tests/Cms/Routes/RouterTest.php

```diff
@@ -294,8 +294,8 @@ public function testDisabledPanel()
             ]
         ]);
 
-        $this->assertNull($app->call('panel'));
-        $this->assertNull($app->call('panel/something'));
+        $this->assertFalse($app->call('panel'));
+        $this->assertFalse($app->call('panel/something'));
```

Comment:
This comes from `Panel::router()`, right? Is there anything that speaks against returning `null` in the error case like before?

Reply:
✅